service/src/main/java/io/camunda/service/license/LicenseType.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.service.license;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+public enum LicenseType {
```

Comment:
❓ Are these the duplicates you mentioned here `Once Optimize is a part of the single application, these duplicate Optimize classes can be removed, and the monorepo implementation can be used` ? So basically both are our addition, but rn we need a temporary backup in optimize too?

Reply:
The `service` implementations are the ones that should be kept around in the future, the Optimize implementations are the duplicates that will be removed when Optimize is placed in the single application architecture